packages/blog-starter-kit/themes/hashnode/components/analytics.tsx

```diff
@@ -4,18 +4,20 @@ import { v4 as uuidv4 } from 'uuid';
 
 import { useAppContext } from './contexts/appContext';
 const GA_TRACKING_ID = 'G-72XG3F8LNJ'; // This is Hashnode's GA tracking ID
+const HASHNODE_ANALYTICS_URL = 'https://user-analytics.hashnode.com/api/analytics'; // This is Hashnode's internal analytics URL
 const isProd = process.env.NEXT_PUBLIC_MODE === 'production';
 const BASE_PATH = process.env.NEXT_PUBLIC_BASE_URL || '';
 
 export const Analytics = () => {
-	const { publication, post } = useAppContext();
+	const { publication, post, series, page } = useAppContext();
 
 	useEffect(() => {
 		if (!isProd) return;
 
 		_sendPageViewsToHashnodeGoogleAnalytics();
 		_sendViewsToHashnodeInternalAnalytics();
 		_sendViewsToHashnodeAnalyticsDashboard();
+		_sendViewsToAdvancedAnalyticsDashboard();
```

Comment:
I get an error if I remove the `!isProd` check:

![image](https://github.com/Hashnode/starter-kit/assets/7524745/d2419374-b5c9-433b-a645-450d4cafa3f3)

Imo all the function should be defined outside of the component and accept arguments instead. That should fix the error.

Reply:
Moving useEffect at end of the component should also fix the error. const and let defined function have temoral dead zone, that's why the reference error.